docs/guide/005-principles-of-metadata-management-in-the-wis.adoc

```diff
@@ -17,5 +17,5 @@ How to publish metadata
 
 Note: For a limited number of records, it is also possible to use the online editing services of a GISC.
 
-For more comprehensive information regarding the WIS and publishing metadata on the WIS, please consult the Manual on WIS (https://wis.wmo.int/WIS-Manual).
+For more comprehensive information regarding the WIS and publishing metadata on the WIS, please consult the Manual on WIS (https://community.wmo.int/activity-areas/wis/wis-guides).
```

Comment:
@amilan17 please check link - thx!

Reply:
this is the official link to the Manual on WIS: https://library.wmo.int/index.php?lvl=notice_display&id=9254